res/css/views/voip/CallView/_CallViewButtons.scss

```diff
@@ -16,6 +16,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// data-whatintent makes more sense here semantically but then the tooltip would stay visible without the button
+// which looks broken, so we match the behaviour of tooltips which is fine too.
+[data-whatinput="mouse"] .mx_CallViewButtons.mx_CallViewButtons_hidden {
+    opacity: 0.001; // opacity 0 can cause a re-layout
```

Comment:
does this cause issues in Firefox? It's been known to disagree with floats on some properties, particularly at low numbers like this.

Reply:
No idea, I'm maintaining the current impl, just tweaking when its shown/hidden